src/lib/draw/Toolbox.svelte

```diff
@@ -79,18 +112,20 @@
 
 <div class="toolbox">
   <div>
-    <Button
-      {mode}
+    <SelectToolButton
       thisMode="edit-attribute"
       label="Edit attributes"
       icon={editAttributesIcon}
       {changeMode}
     />
-    <AttributeMode bind:this={attributeMode} {mode} {changeMode} />
+    <AttributeMode
+      bind:this={attributeMode}
+      {changeMode}
+      bind:eventHandler={eventHandlers["edit-attribute"]}
```

Comment:
https://svelte.dev/tutorial/text-inputs
I think `bind:` is only if `AttributeMode` was overwriting the thing passed in and we want it to flow back up. It's not; it's just mutating it. The pointer to `eventHandlers["edit-attribute"]` shouldn't change, so I don't think we need it